components/alibi-explain-server/pyproject.toml

```diff
@@ -6,19 +6,21 @@ authors = ["Seldon Technologies Ltd. <hello@seldon.io>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.7"
-alibi = "0.6.2"
+python = "^3.7,<3.8"
```

Comment:
```suggestion
python = "~3.7"
```

Reply:
True, would work the same but if that's the only change I'd rather skip resolving lock file again and need to run the suite of tests :)